.gitignore

```diff
@@ -20,3 +20,5 @@ Gemfile.lock
 package-lock.json
 yarn.lock
 *.log
+*.swp
+
```

Comment:
Definitely not intended change

Reply:
I will try to fix those and do another commit.

Reply:
> However I wonder about the .gitignore chage. *.swp files are part of vim temporary swap files and might be beneficial to include those in .gitignore

Feel free to propose it in separate PR.

> I will try to fix those and do another commit.

Please note that I haven't got any hardware to test new function, and I didn't review your implementation.
May be better to include this gesture support into `Map.ScrollWheelZoom.js` or `Map.TouchZoom.js` directly (I cannot say now).

In any case I do not feel proposed handler's name good, as term "gesture" has too broad meaning.

Reply:
I though about extending ScrollWheelZoom because touchpad gestures effectively emulate a scroll wheel at the macos level, but this is not about zooming anymore but panning too and at the user level, it is a touchpad gesture.